guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -19,6 +19,55 @@ The advantage of this is that each organization maintains separate subscriptions
 You can use future-dated subscriptions in a subscription manifest.
 When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
+.Benefits of the subscriptions service
```

Comment:
This title doesn't make it clear at all that we're now talking about the Subscriptions service of console.redhat.com, rather than subscriptions that you add to your manifest. I'm a bit confused why this section is being added here.

Reply:
I'm renaming it to _Subscriptions service capabilities_.